src/ontology/fbbt-edit.obo

```diff
@@ -83176,41 +83330,41 @@ creation_date: 2018-10-23T09:02:40Z
 
 [Term]
 id: FBbt:00047867
-name: sensillum campaniformium of dorsal radius D
+name: campaniform sensillum of dorsal radius D
 def: "Sensillum belonging to the anterior group of campaniform sensilla of the dorsal surface of the medial radius. There are four of these circular sensilla, which have a high profile and a socket (type 1). Each sensillum is approximately 4.5 micrometers in diameter." [FlyBase:FBrf0038304]
-synonym: "campaniform sensillum of dorsal radius D" EXACT [FlyBase:FBrf0038304]
 synonym: "d.Rad.D" EXACT [FlyBase:FBrf0038304]
-is_a: FBbt:00004346 ! sensillum campaniformium of dorsal radius Sc12
+synonym: "sensillum campaniformium of dorsal radius D" EXACT [FlyBase:FBrf0038304]
```

Comment:
not sure if this reference is still appropriate - if they were calling it "campaniform sensillum of dorsal radius D", they maybe weren't calling it "sensillum campaniformium of dorsal radius D", so probably best to get rid of references from newly added synonyms.

Reply:
The reference is a paper from 1982 so there's little hope of checking the full text, but the available abstract uses "campaniform", so I agree it's not relevant to cite them as reference for a synonym they were likely not using at all.